package.json

```diff
@@ -17,13 +17,13 @@
     ]
   },
   "dependencies": {
-    "@hapi/boom": "9.x.x",
-    "@hapi/hoek": "9.x.x"
+    "@hapi/boom": "10.x.x",
+    "@hapi/hoek": "10.x.x"
   },
   "devDependencies": {
-    "@hapi/code": "8.x.x",
+    "@hapi/code": "9.x.x",
```

Comment:
Since we're updating the version, should we move it to npm's default range management aka caret?